helmfile/values/networkpolicy/service-cluster.yaml.gotmpl

```diff
@@ -74,29 +74,27 @@ fluentd:
 
 rcloneSync:
   enabled: {{ and .Values.objectStorage.sync.enabled .Values.networkPolicies.rcloneSync.enabled }}
-  {{- if and .Values.objectStorage.sync.enabled .Values.networkPolicies.rcloneSync.enabled }}
   destinationObjectStorage:
     ips:
-      {{- if and (hasKey .Values.objectStorage.sync.s3 "regionEndpoint" ) (.Values.objectStorage.sync.s3.regionEndpoint) (.Values.networkPolicies.rcloneSync.destinationObjectStorageS3.ips) }}
+      {{- if eq .Values.objectStorage.sync.destinationType "s3" }}
```

Comment:
All the changes in this file doesn't take into account the buckets type from ```objectStorage.sync.bucket.destinationType``` 
only the```objectStorage.sync.destinationType```  if the bucket type is opposite of the sync destinationType its ips/ports won't be included because its not included in these if statements.

Maybe do something like the [sourceSwift](https://github.com/elastisys/compliantkubernetes-apps/blob/e08d35d7a4a7cbe875ddc668b4d09a12dbd46d89/helmfile/values/networkpolicy/service-cluster.yaml.gotmpl#L12-L20) has but for both s3 and swift.

Reply:
added I need to test it